### PR TITLE
gui/settings_dialog: Update lang strings and adjust debug options.

### PR DESCRIPTION
**Pull request by Croden1999**
- gui/settings_dialog: Update lang strings and adjust debug options.
- gui/content_manager: Fixed small issue about missing addcont (DLC) title. 
- lang: Fixed pl.xml and ms.xml load error.

### CODE REVIEW

**Comment by Kitakatarashima** [comment]
This is the fix for Indonesian where some dialogues are missing and fixes how many translation errors there are in Indonesian and Malaysian bro @Macdu , please add new dialogue in the file is if you want.
[language updates and fixes, Indonesian and Malaysian.zip](https://github.com/Vita3K/Vita3K/files/13816196/language.updates.and.fixes.Indonesian.and.Malaysian.zip)

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -831,7 +831,9 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::Spacing();
         ImGui::Separator();
         ImGui::Spacing();
-        if (ImGui::Combo(lang.emulator["log_level"].c_str(), &emuenv.cfg.log_level, "Trace\0Debug\0Info\0Warning\0Error\0Critical\0Off\0"))
+
+        static const char *LIST_LOG_LEVEL[] = { lang.emulator["trace"].c_str(), "Debug", lang.emulator["info"].c_str(), lang.emulator["warning"].c_str(), lang.emulator["error"].c_str(), lang.emulator["critical"].c_str(), lang.emulator["off"].c_str() };
```
Comment (Macdu) [inline vita3k/gui/src/settings_dialog.cpp:835]: Don't use static, it means if you change the lang it won't be modified

Comment (Croden1999) [reply vita3k/gui/src/settings_dialog.cpp:835]: i got it, thanks, sir.

**Comment by Croden1999** [comment]
ready for review

**Review by Macdu — APPROVED** [review]

**Comment by Macdu** [comment]
@Croden1999 There is a conflict now, could you resolve it ? (sorry about that)

**Comment by Croden1999** [comment]
> @Croden1999 There is a conflict now, could you resolve it ? (sorry about that)

okay, sir.

File: vita3k/gui/src/gui.cpp [hunk 2]
```diff
@@ -609,6 +609,7 @@ void get_user_apps_title(GuiState &gui, EmuEnvState &emuenv) {
 }
 
 void get_sys_apps_title(GuiState &gui, EmuEnvState &emuenv) {
+    auto &lang = gui.lang.sys_apps_title;
```
Comment (Macdu) [inline vita3k/gui/src/gui.cpp:612]: You are not using it

Comment (Croden1999) [reply vita3k/gui/src/gui.cpp:612]: it is used only not installed firmware

**Review by Macdu — APPROVED** [review]

**Pull request merged**